src/net.h

```diff
@@ -284,7 +284,7 @@ class CBanEntry
     ADD_SERIALIZE_METHODS;
 
     template <typename Stream, typename Operation>
-    inline void SerializationOp(Stream& s, Operation ser_action, int nType, int nVersion) {
+    inline void SerializationOp(Stream& s, Operation ser_action, int nType, int _nVersion) {
```

Comment:
I don't see what this was supposed to be doing before, but this look like a self-assign now.